scripts/process/hydrographs.R

```diff
@@ -0,0 +1,16 @@
+process.hydrographTotal <- function(viz, smooth.days){
```

Comment:
I know this part is a work in progress, but I'd make smooth.days a variable in viz